src/clr-core/common/utils/enums.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export enum KeyCodes {
+  ArrowLeft = 'ArrowLeft',
+  ArrowUp = 'ArrowUp',
+  ArrowRight = 'ArrowRight',
+  ArrowDown = 'ArrowDown',
+  Backspace = 'Backspace',
+  Tab = 'TAB',
+  Enter = 'Enter',
+  Escape = 'Escape',
+  Space = 'Space',
+}
+
```

Comment:
We have this file: https://github.com/vmware/clarity/blob/master/src/clr-angular/utils/key-codes/key-codes.ts

Wondering if there's opportunities to consolidate or reuse? I also think there was minor differences in KeyCodes and/or Key depending on the browser. I was trying to find a specific example of checking for that in our code base, but maybe this can be discovered as we go along. I'll post a link if I find it.

Reply:
Completely forgot about the existing ones, thanks! We also have https://github.com/vmware/clarity/blob/master/src/clr-angular/utils/focus/key-codes.enum.ts 
I'll update to reuse the existing ones. I remember the browser difference. In IE the keycodes are different so I'll need to update to check for that.

Reply:
Cleaned up the enum version of the duplicate in clr-core and clr-angular